src/main/java/org/broadinstitute/hellbender/utils/pairhmm/PairHMM.java

```diff
@@ -35,7 +36,18 @@
         /* PairHMM as implemented for the UnifiedGenotyper. Uses log10 sum functions accurate to only 1E-4 */
         ORIGINAL(() -> new Log10PairHMM(false)),
         /* Optimized version of the PairHMM which caches per-read computations and operations in real space to avoid costly sums of log10'ed likelihoods */
-        LOGLESS_CACHING(() -> new LoglessPairHMM());
+        LOGLESS_CACHING(() -> new LoglessPairHMM()),
+        /* Optimized AVX implementation of LOGLESS_CACHING called through JNI */
+        VECTOR_LOGLESS_CACHING(() -> {
```

Comment:
As we discussed before, I think we should have a `FASTEST_AVAILABLE` option that tries to use the native code and falls back to the pure java if avx fails. Specifying AVX if it isn't available should fail so that people can detect problems with their systems.